docs/config-file/v2.rst

```diff
@@ -318,6 +321,25 @@ as defined here:
 * `stable <https://github.com/readthedocs/readthedocs-docker-images/tree/releases/5.x>`_: :buildpyversions:`stable`
 * `latest <https://github.com/readthedocs/readthedocs-docker-images/tree/releases/6.x>`_: :buildpyversions:`latest`
 
+build.apt_packages
+``````````````````
+
+List of `APT packages`_ to install.
```

Comment:
```suggestion
List of `APT packages`_ to install.
Our build servers run Ubuntu, with the default set of package repositories installed.
We don't currently support PPA's or other custom repositories. 
```

Reply:
I would actually link here to the versions of Ubuntu that our build servers run based on the settings the user selected. Packages change names between versions of Ubuntu or could be added/removed in specific versions. We might consider noting that folks shouldn't use this for installing binary versions of Python packages (eg. `python3-numpy`)

Reply:
Updated the docs with those notes :+1: